CHANGELOG

```diff
@@ -52,6 +52,10 @@ Miscellaneous
 
 - Add support for building a shared version of the MFEM library with GNU make.
 
+- In FiniteElementSpace, the fully assembled refinement matrix is now replaced
```

Comment:
Revise, e.g. "... is now replaced by default by... . The fully assembled matrix is still available if ..."